.github/workflows/main.yml

```diff
@@ -3,11 +3,38 @@ name: CI
 on: [pull_request, push]
 
 jobs:
+  msrv:
+    name: MSRV
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: 1.34.0
+          override: true
+
+      - name: Check
+        uses: actions-rs/cargo@v1
+        with:
+          command: check
+
   check:
     name: Check
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          command: check
+          toolchain: "1.34.0"
+          override: true
+
```

Comment:
Is there a reason to have the check job run on 1.34 when there's already a separate MSRV job?

Reply:
Nope, I forgot to remove it. Good catch!